app/controllers/concerns/vet360/writeable.rb

```diff
@@ -39,7 +39,13 @@ def build_record(type, params)
     end
 
     def validate!(record)
-      raise Common::Exceptions::ValidationErrors, record unless record.valid?
+      unless record.valid?
```

Comment:
I'm surprised rubocop didn't say anything here. This should fail the [guard clause cop](https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/GuardClause).

Should be:
```
return if record.valid?

# code
```